exist-versioning-release.md

```diff
@@ -208,101 +313,169 @@ You will require a system with:
     What is the new development version for "eXist-db"? (org.exist-db:exist) 5.4.0-SNAPSHOT: :
     ```
 
-6.  Once the prepare process completes you can perform the release. This will upload Maven Artifacts to Maven
-Central (staging), Docker images to Docker Hub, and eXist-db distributions and installer to Github releases:
-    ```
-    $ mvn -Ddocker=true -Dmac-signing=true -Djarsigner.skip=false -Darguments="-Ddocker=true -Dmac-signing=true -Djarsigner.skip=false" release:perform
-    ```
+5. Test the release artefacts as much as possible on different Operating systems (Mac, Windows, Linux).  
+
+   * `$EXIST/exist-distribution/target/` 
+     * eXist-db-<version>.dmg
+     * exist-distribution-eXist-db-<version>.bz2
+     * exist-distribution-eXist-db-<version>-win.zip
+   * `$EXIST/exist-installer-<version>.jar` 
+
+#### Signing Binaries (work in progress)
+
+In order to allow end-users to run the installer as well as the application on macOSX (Catalina) the application needs to be notarized by Apple.
```

Comment:
Reword this to something like:
```
In order to allow end-users to open the DMG, and then later the eXist-db application on macOS without any security blocks, both the application and the DMG need to be signed, notarised by Apple, and then stapled.
```

Reply:
Building on Adam's suggestions, I've added some more context and a link:

> In order to allow users of macOS 10.13 (Catalina) and higher to launch the eXist installer and application with minimal security warnings from the operating system, Apple encourages developers to code-sign their applications with an Apple Developer ID and submit the application (and disk image, or DMG, in the case of applications shipped in a DMG) to Apple's [notarization service](https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution). The notarization service scans the application and the DMG for malicious software, and returns a certificate that the developer can staple to the application and DMG before a release is distributed to users. Without preparing eXist releases this way, users will encounter security warnings that require multiple steps to work around. Thus, in the eXist release process, both the application and the disk image (DMG) needs to be signed with an Apple Developer ID, notarized by Apple, and the notarization ticket need to be stapled to each. 

(It's macOS, not macOSX.)